osler/workup/models.py

```diff
@@ -130,17 +131,13 @@ def group_can_sign(cls, group):
         return group_has_perm(group, cls.get_sign_perm())
 
 
+class AbstractBasicNote(Note):
```

Comment:
I don't get why we need an AbstractBasicNote when Note is already abstract. The new fields it defines aren't very important or plentiful so I think it's ok to repeat in the children models. Having an additional abstract note feels messier to me.

Reply:
That was my thought too at first. What happens when AttestableBasicNote inherits from BasicNote is that pt.basicnote_set includes their attestablebasicnote_set, and pt.attestablebasicnote_set does not exist. Wanted to avoid that, and the issue seemed to be that Django treats children of concrete parent model as that parent model

Reply:
Correction: I do get why and I'm ok with it if you like it, I still prefer not

Reply:
Hmmm I meant AttestableBasicNote and BasicNote both just inheriting from Note, and just defining title and text for both of them... not the most elegant I know. On further consideration, I'm ok with it this way, maybe one day in the future we will add more types of free text boxes and AbstractBasicNote will feel more useful lolol.